src/app/services/configuration-control.service.ts

```diff
@@ -398,7 +404,7 @@ export class ConfigurationControlService {
       inputStyle: 'textarea',
       autoRefresh: true
     };
-    if (type === 'JSON') {
+    if (type === 'JSON' || type === 'list' || type === 'kvlist') {
```

Comment:
`if (type === 'JSON' || type === 'list' || type === 'kvlist')` 
can be
`if(['JSON','list','kvlist').includes(type))`